extensions/script-libraries/extension-utils/extension-utils.livecodescript

```diff
@@ -806,8 +806,9 @@ private command __extensionBuildPackage pDetailsA, pFolder, pTargetFolder, pRemo
          tSourceFile, tSupportFiles, pFolder, tArchive, \
          pDetailsA["module_files"], pDetailsA["icon"], \
```

Comment:
@livecodeali what do you think about just passing `pDetailsA` here... looks like it would be easy to not notice a typo here. We should probably have a policy to use constants for array keys to reduce typo risk also.

Reply:
That's a good point, yeah I will do that